tests/conftest.py

```diff
@@ -1904,6 +1905,26 @@ def _awscli_pod_cleanup():
     return awscli_pod_obj
 
 
+@pytest.fixture()
+def test_directory_setup(request, awscli_pod_session):
+    return test_directory_setup_fixture(request, awscli_pod_session)
+
+
+def test_directory_setup_fixture(request, awscli_pod_session):
+    origin_dir, result_dir = setup_pod_directories(
+        awscli_pod_session, ["origin", "result"]
+    )
+    SetupDirs = namedtuple("SetupDirs", "origin_dir, result_dir")
+
+    def dir_cleanup():
```

Comment:
similar to https://github.com/red-hat-storage/ocs-ci/pull/3754/files#diff-cdf7f325cc2d8ea0bfeaffe360e4a13f9c69bf0534861e737af7946520d27e67R2006

Suggest creating a common function that returns the test name